app/controllers/admin/embeddable_media_controller.rb

```diff
@@ -0,0 +1,79 @@
+class Admin::EmbeddableMediaController < Admin::AdminController
+
+  def new
+    owner_type = params[:owner_type]
+    owner_id = params[:owner_id]
+    owner_attribute = params[:owner_attribute]
+    @record = EmbeddableMedia.new(owner_type: owner_type, owner_id: owner_id, owner_attribute: owner_attribute)
+    # need this until migration updated and all developers have newly migrated data
+    handle_authorize
+    render 'linked_sheet'
+  end
+
+  def create
+    @record = EmbeddableMedia.new(record_params)
+    handle_authorize
+
+    @record.parse_key_gid_from_original_url
+    if @record.save
+      @item_owner = @record.owner
+      if @item_owner[:custom_value_set_linkable_type] == "Loan"
+        @loan = Loan.find(@item_owner[:custom_value_set_linkable_id])
+        redirect_to admin_loan_path(@loan, anchor: 'questions'), notice: I18n.t("linked_sheet.notice_saved")
```

Comment:
Flagging this for review later: I'm surprised this is redirecting as it implies this request is not running over AJAX, which makes me wonder how this works if someone is in the middle of filling out the form. Is it an iframe? I guess I'll find out...